Procurement/ViewModel/Recipes/Recipe.cs

```diff
@@ -38,5 +39,13 @@ public virtual IEnumerable<RecipeResult> Matches(IDictionary<Tab, List<Item>> it
             }
             return Matches(flatItems);
         }
+
+        protected bool GetShouldUseShortRecipeDescriptions()
+        {
+            bool result = false;
+            bool.TryParse(Settings.UserSettings.GetEntry("UseShortRecipeDisplayDescriptions"), out result);
```

Comment:
Beyond the scope of this PR, but settings are in dire need of a rework. Needing to stuff like this is just horrible.